pkg/cmd/pulumi/convert/convert_test.go

```diff
@@ -48,12 +49,12 @@ func TestYamlConvert(t *testing.T) {
 		t.Fatalf("Pulumi.yaml is a directory, not a file")
 	}
 
-	cwd, err := filepath.Abs("testdata")
+	cwd, err := filepath.Abs("convert_testdata")
 	require.NoError(t, err)
 
 	result := runConvert(
-		pkgWorkspace.Instance, env.Global(), []string{}, cwd, []string{},
-		"yaml", "go", "testdata/go", true, true, "")
```

Comment:
I think this is a bad conflict merge -- you want `testdata` on your side here I think since that got moved under `convert/`.